projects/USU_Kaundal.yaml

```
@@ -0,0 +1,11 @@
+Description: Biological data is accumulating faster than people’s capacity to analyze them. Our research interests and goals revolve around mitigating this issue in the context of “information to inference” scope. At USU, Dr. Kaundal has developed an independent and collaborative research program in bioinformatics, primarily focusing on computational mining of large multi-dimensional -omics datasets, and computational modeling using supervised (Machine Learning) and unsupervised (Bayesian-based) learning. Our group is actively developing novel tools and software to apply the gained knowledge towards organismal improvement. Research in KAABiL laboratory generally falls under the following major program objectives: http://bioinfo.usu.edu/research
```

```suggestion
Description: >-
  Biological data is accumulating faster than people’s capacity to analyze them. Our research interests and goals revolve around mitigating this issue in the context of “information to inference” scope. At USU, Dr. Kaundal has developed an independent and collaborative research program in bioinformatics, primarily focusing on computational mining of large multi-dimensional -omics datasets, and computational modeling using supervised (Machine Learning) and unsupervised (Bayesian-based) learning. Our group is actively developing novel tools and software to apply the gained knowledge towards organismal improvement. Research in KAABiL laboratory generally falls under the following major program objectives: http://bioinfo.usu.edu/research
```
Try this? `>-` is one of the ways to denote a YAML string; it's useful because you don't have to worry about quoting, you just have to indent the rest of the string.

---

https://yaml-multiline.info/ is a good resource

---

As a side note, yes, someone has and is presumably paying money every year for a website that has the sole purpose of explaining to people how to write multiline strings in YAML.  I apologize for our choice of data format.